account-integrations/safe/src/SafeBlsPlugin.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+pragma abicoder v2;
```

Comment:
Seen that this is included in the `SafeECDSAPlugin` too. I believe abicoder was defaulted to v2 as of [solidity 0.8.0](https://docs.soliditylang.org/en/v0.8.21/080-breaking-changes.html#silent-changes-of-the-semantics). I think it makes sense here and in the ECDSA plugin as we're using solidity`>=0.7.0`. Curious if we want to keep supporting versions of solidity less than 0.8.0? Open question don't really have a strong opinion

Reply:
Sorry just getting around to these comments.  Yeah, I don't have a preference either.  I haven't really done a deep dive to know if it's still necessary to support the older versions